### PR TITLE
acq millingmng: after moving to a site, reset the beam shift

### PR DESCRIPTION
**Pull request by pieleric**
The beam shift is used to compensate for the drift while milling.
Every time we move to a new place on the sample, we have a "fresh canvas",
so we should reset the beam shift. One of the advantage being that it
reduces the chance of getting out of the limits of the shift during
milling.
In addition, we can use the beam shift to compensate for the
impressision of the stage movement.
Of course, there might still have been a drift between the moment the
feature was selected, and the milling, but for now we don't handle that.

### CODE REVIEW

**Review by tepals — APPROVED** [review]

**Comment by pieleric** [comment]
Let's wait for Wilco to test it on the real hardware before we merge.

File: src/odemis/acq/millmng.py
Comment (pieleric) [reply src/odemis/acq/millmng.py]: When a move is requested, the motors go towards that direction, and the encoder reads the position. With close-loop control, eventually the motor reaches a position *near* where you've asked. The motor precision is usually lower than the encoder precision, so it rarely reaches exactly the target position. It's interesting to know the difference. That said, on the MIMAS stage, it seems that the encoder also doesn't have a precision as good as the user would want (< 10nm), so we might need other techniques to improve the position.

**Review by K4rishma — APPROVED** [review]

**Review by tepals — APPROVED** [review]

**Pull request merged**
